CHANGES.rst

```diff
@@ -172,6 +172,12 @@ Bug Fixes
 
 - ``astropy.utils``
 
+  - Fix to allow the C-based _fast_iterparse() VOTable XML parser to
+    relloc() its buffers instead of overflowing them.
+    (Bug occuring when Gzip-compressed VOTable XML streams are passed to
```

Comment:
Sorry, what I meant is indeed to drop the second sentence, but you've already done that.